deployment/templates/mongodb/mongodb-pvc.yaml

```diff
@@ -0,0 +1,28 @@
+{{ if .Values.volumes.mongodb.deployLocalPv }}
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+ name: mongo-pv
+ labels:
+   type: local
+spec:
+ storageClassName: {{ .Values.volumes.mongodb.storageClass }}
+ capacity:
+   storage: {{ .Values.volumes.mongodb.size }}
+ accessModes:
+   - ReadWriteOnce
+ hostPath:
+   path: {{ .Values.pathToLocalDir }}
+{{ end }}
```

Comment:
PVs will probably not work in OpenShift. They are created automatically when the PVC is created.

Reply:
But as Kaniko does not currently work in Openshift, it is not a very important issue.

Reply:
At the moment you can set the deploy PV value to false and it won't get deployed. Instead you can use an existing RWO StorageClass for the PVC.

Reply:
👍🏽